packages/shared/src/config/modules.ts

```diff
@@ -18,6 +18,11 @@ export const getDefaultModuleInfo = (name: string, path: string): ModuleInfo =>
 
 export const epic: ModuleInfo = getDefaultModuleInfo('epic', 'epics/ContributionsEpic');
 
+export const epicWithChoiceCards: ModuleInfo = getDefaultModuleInfo(
```

Comment:
Because the variation is determined by the `variant` settings, I don't think we need a separate module for this test. So this can be removed.
The existing `ContributionsEpic` module has the new behaviour

Reply:
👍🏼